api/src/app.ts

```diff
@@ -105,14 +106,43 @@ app.use(function (req, res) {
   res.status(500).json({ error: `Internal Serverless Error - '${req}'` });
 });
 
+let conn: null | Mongoose = null;
+const uri = process.env.MONGO_URL;
+export const connect = async () => {
+  if (conn == null) {
+    conn = await mongoose.connect(uri, {
+      dbName: DB_NAME,
+      serverSelectionTimeoutMS: 5000,
+    });
+  }
+  return conn;
+};
+app.use(async function (req, res, next) {
+  await connect();
+  next();
+});
```

Comment:
I don't think we need this middleware since we connect on lambda setup or before starting the dev server.

```suggestion
```